src/scripts/extract-csm-abi.ts

```diff
@@ -20,9 +20,10 @@ const PARTS = [
   'NodeOperatorAdded',
   'RESUME_ROLE',
   'resume',
+  'NodeOperatorRewardAddressChanged',
 ];
 export const getCSMAbi = () => {
-  const { abi } = getCSMArtifact();
+  const abi = getCSMArtifact();
```

Comment:
How did it work previously with  { abi } ?

Reply:
because it worked with the result of `just build`
but now I'm using
```
just abis
just interfaces
```
because before there were not enough events